### ISSUE TITLE
how to save txt file at youtube chat ?

### ISSUE THREAD

**Issue by sherlyangelia** [xenova/chat-downloader#93]
Hi,

How to save txt file from youtube live, such as :

[Time] | [Name] : [Message]

Not like json format like this : 
[
    {
        "action_type": "add_banner_to_live_chat",
        "author": {
            "badges": [
                {
                    "icon_name": "verified",
                    "title": "Verified"
                },
                {
                    "icon_name": "owner",
                    "title": "Owner"
                }
            ],

Thanks :)

**Comment by Felk** [comment]
Can you provide the command you executed?

The format should get auto-detected by the file extension, so `--output chat.json` outputs JSON, and `--output chat.txt` outputs text. However, the text format is not the same text format you see printed on the console, but python representations. Also see issues https://github.com/xenova/chat-downloader/issues/77 and https://github.com/xenova/chat-downloader/issues/78 for more info on that

**Comment by xenova** [comment]
This has been implemented in the latest version (v0.1.0). The `--output` parameter accepts a file name and now the chat is saved in the way it is output (as requested). For example, running:

```
chat_downloader https://www.youtube.com/watch?v=5qap5aO4i9A --output text.txt
```

will write the chat to `text.txt` in the following format:
```
2021-06-30 20:15:56 | 杜睿悠: Which city?
2021-06-30 20:15:56 | Anna Luiza Ferreira Santos da Hora: só inscrita nova😊
2021-06-30 20:15:57 | kura chyn: der aufkärer lol
2021-06-30 20:15:59 | Steve Ski: Лаптоп Ноутбуков its an Mobile Game
2021-06-30 20:16:01 | Pattonized: Deutsche hier???
2021-06-30 20:16:01 | Vinicius Brambilla: @Ana DLucca cria ai man pfv
2021-06-30 20:16:04 | Emely Wienand: hier sind voll viele deutsche
2021-06-30 20:16:06 | Ignacio Agustín Martínez: Algien de argentina?
```

Let me know if you have any other questions 👍
